content/en/tracing/llm_observability/quickstart.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Quickstart
+---
+<div class="alert alert-info">LLM Observability is in public beta.</a></div>
+
+Our quickstart docs make use of our Python SDK. For detailed usage, see [the SDK documentation][1]. If your application is written in another language, you can create traces by calling the [API] instead.
+
+## Jupyter notebook quickstart
+
+To try out LLM Observability in a series of guided notebooks, follow the instructions in [the LLM Observability onboarding repo][2].
+
+## Command line quickstart
+
+Use the steps below to run a simple Python script that generates an LLM Observability trace.
+
+### Prerequisites
+
+- LLM Observability requires a Datadog API key (see [the instructions for creating an API key][7]).
```

Comment:
Should we also say Datadog site here as well?

Reply:
I know we have 
Replace `<YOUR_DATADOG_SITE>` with the Datadog site you want to send data to, such as `datadoghq.com` or `us3.datadoghq.com`.

So either we can move `DD_API_KEY` requirement down in the same place as datadog site, or move datadog site up to this level of the doc?

Reply:
@lievan, the key is listed here because it's a process the user has to go through (if they don't already have one). The Datadog site is just information about them, no setup required, so args like that are generally not listed as prerequisites. Similarly, in the Quickstart prereqs, we list the Datadog key and the OpenAI key because those have to be set up.